src/webview/__tests__/generateInboundEventEditSequence-test.js

```diff
@@ -198,7 +198,9 @@ const pmMessages6 = [
 const baseState = eg.reduxStatePlus({
   streams: [...eg.plusReduxState.streams, stream1, stream2],
```

Comment:
This could also collide. I'm not sure what the consequences would be -- it might be that the later entry wins, in which case it's a purely latent bug (but a bug because our `streams` array isn't supposed to have duplicate stream IDs.)